packages/tooltip/components/TooltipIcon.tsx

```diff
@@ -8,7 +8,7 @@ interface TooltipIconProps extends Omit<IconProps, "shape"> {
   /**
    * Icon to display, choose either (i) or (?).
    */
-  shape?: SystemIcons.CircleInformation | SystemIcons.CircleQuestion;
+  shape?: string;
```

Comment:
This one we definitely don't want to change because we are explicitly expecting either of these shapes for a tooltip icon.